### DISCUSSION TITLE
immich docker creates anonymous volume

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#9240] [General]
Every time I start the docker container with pretty much the default docker file I get a new anonymous volume created, mounted in /data under /immich_redis. It seems to never contain any data. I have no unlabeled volumes in my compose file, so I wonder how to turn that off.

Str1atum

**Comment by mmomjian** [comment]
If you want you can mount this folder to a bind mount on your filesystem. Otherwise the anonymous volume will be created.

**Comment by 8MadHatter231** [comment]
I know this is old but here's the solution:
Add a named volume to Redis in your docker-compose:

redis:
  container_name: immich_redis
  image: docker.io/valkey/valkey:8
  **volumes:
    - redis-data:/data**
  healthcheck:
    test: redis-cli ping || exit 1
  restart: always

And add to the bottom:

volumes:
  model-cache:
  **redis-data:**
